Cython/Compiler/PyrexTypes.py

```diff
@@ -3876,6 +3878,104 @@ def check_nullary_constructor(self, pos, msg="stack allocated"):
         if constructor is not None and best_match([], constructor.all_alternatives()) is None:
             error(pos, "C++ class must have a nullary constructor to be %s" % msg)
 
+class ScopedEnumType(CType):
```

Comment:
Seems worth naming this `CppScopedEnumType` to make it stick out in the code as something C++  specific.

Reply:
Done